examples/nft-nd-nns/go.mod

```diff
@@ -4,6 +4,6 @@ go 1.16
 
 require (
 	github.com/nspcc-dev/neo-go v0.98.2-pre.0.20220318104433-7a8dc1179138
```

Comment:
Also need to update `neo-go` revision in the next commit.

Reply:
OK.